### DISCUSSION TITLE
Sqlite SEE

### DISCUSSION THREAD

**Discussion by neervanbiqs** [pawelsalawa/sqlitestudio#4877] [Q&A]
I have a database that was encrypted with official sqlite SEE. I know its password and can open it with Sqlite Manager with builtin sqlite SEE.
But I can to do the same with SqliteStuido. What parameters (pragma) should I specify to decrypt a database encrypted with an aes128 key?

**Comment by pawelsalawa** [comment]
SQLiteStudio does not support SEE.

**Comment by samsharp99** [comment]
Could support be added (perhaps by allowing us to configure our own SQLite DLL path)?
